disnake/automod.py

```diff
@@ -627,7 +645,7 @@ def __init__(self, *, data: AutoModActionExecutionEventPayload, guild: Guild) ->
         self.channel_id: Optional[int] = _get_as_snowflake(data, "channel_id")
         self.message_id: Optional[int] = _get_as_snowflake(data, "message_id")
         self.alert_message_id: Optional[int] = _get_as_snowflake(data, "alert_system_message_id")
-        self.content: str = data["content"]
+        self.content: str = data.get("content") or ""
```

Comment:
Why not the following?
```diff
-        self.content: str = data.get("content") or ""
+        self.content: str = data.get("content", "")
```

Reply:
Primarily so that `{"content": null}` will still fall back to `""` - seeing how these are apparently subject to arbitrary changes I don't really trust the API docs on the non-nullability of this field.

Reply:
Sounds good